### PR TITLE
Post-install step to create cluster SNAT policy

### PR DESCRIPTION
**Pull request by apoorva11029**
Add snat IP and kubeconfig path to inventory inside aci_cni field like
aci_cni:
  cluster_snat_policy_ip:
  kubeconfig:

### CODE REVIEW

File: upi/openstack/inventory.yaml [hunk 1]
```diff
@@ -34,6 +34,7 @@ all:
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
       aci_cni:
+        snatIP: 10.0.0.0/21
```
Comment (snaiksat) [inline upi/openstack/inventory.yaml:37]: We have already defined a config for this, see "cluster_snat_policy_ip" further below

File: upi/openstack/post-install.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  tasks:
+  - name: 'Create snat policy file'
+    template:
+      src: cluster_snat_policy.conf.j2
+      dest: cluster_snatpolicy.yaml
+
+  - name: 'Create cluster SNAT policy'
+    k8s:
+      state: present
+      kubeconfig: "{{ kubeconfig }}"
```
Comment (abhis2112) [inline upi/openstack/post-install.yaml:13]: do we have {{kubeconfig}} declared in inventory?

File: upi/openstack/inventory.yaml (hunk 1 above)
Comment (apoorva11029) [reply upi/openstack/inventory.yaml]: Right, fixed!

File: upi/openstack/post-install.yaml (hunk 2 above)
Comment (apoorva11029) [reply upi/openstack/post-install.yaml]: yes, user will have to add it under aci_cni['kubeconfig']

**Pull request merged**